query (Top)Schaufensterkonditionen add validation rules auszahlungsbetrag and laufzeit (#53)
europace/kex-angebote-api: README.md

### Query Top-Schaufensterkondition

#### Hints

> * The calculation of Top-Schaufensterkondition is only possible if there is a valid **Auszahlungsbetrag** and **LaufzeitInMonaten**. 

Reviewer: ```suggestion
* The calculation of Top-Schaufensterkondition requires a valid `auszahlungsbetrag` and `laufzeitInMonaten`.
```

  If the provided values are not valid, an [GraphQL-Error](#graphql-errors) with the status code `400` is returned.

#### Request

The GraphQL-Query is called `topSchaufensterkondition` and contains the following parameter:

> | Parameter Name     | Type                                       | Default Value                                     | Condition      |

Reviewer: Condition is okay, maybe Explanation or Description is more versatile.

|--------------------|--------------------------------------------|---------------------------------------------------|----------------|
| partnerId          | [Partner-ID](#partner-id)                  | The Partner-ID of the API Client                  |                |
| auszahlungsbetrag  | Euro!                                      | - (mandatory field)                               | greater than 0 | 

> | laufzeitInMonaten  | Int                                        |                                                   | greater than 0 |

Reviewer: Even if the datatype is doesn't make it mandatory, we should mark it as mandatory. And I would make the additional text more verbose, just like the rest.
```suggestion
| laufzeitInMonaten  | Int                                        | - (mandatory field)                               | Value must be greater than 0 |
```

### Query Schaufensterkonditionen

#### Hints

> * The calculation of Top-Schaufensterkondition is only possible if there is a valid **Auszahlungsbetrag** and **LaufzeitInMonaten**.

Reviewer: ```suggestion
* The calculation of Top-Schaufensterkondition requires a valid `auszahlungsbetrag` and `laufzeitInMonaten`.
```

  Should that not be the case the API user receives a [GraphQL-Error](#graphql-errors) with the status code `400`.

#### Request

The GraphQL-Query is called `schaufensterkonditionen` and has the following parameter:

| Parameter Name     | Type                                      | Default Value                       |
|--------------------|-------------------------------------------|-------------------------------------|
| partnerId          | [Partner-ID](#partner-id)                 | The Partner-ID of the API Client    |
| auszahlungsbetrag  | Euro!                                     | - (mandatory field, greater than 0) |

> | laufzeitInMonaten  | Int                                       | Duration in month (greater than 0)  |

Author: @sgrell I introduced the column `condition`.

| finanzierungszweck | [Finanzierungszweck](#finanzierungszweck) | `FREIE_VERWENDUNG`                  |                |

> | datenkontext       | [Datenkontext](#datenkontext)             | `TESTUMGEBUNG`                      |                |

Reviewer: see above
